app/components/select-month/template.hbs

```diff
@@ -1 +1 @@
-{{primitive-select name=name update=attrs.update prompt=prompt items=months}}
+{{primitive-select name=name data-stripe=data-stripe update=attrs.update prompt=prompt items=months}}
```

Comment:
Since `{{select-month}}` is a payment only thing ATM, you can omit `name=` here (along with the tweaks mentioned above).